docs/en_US/NAS/one_shot_nas.rst

```diff
@@ -1,19 +1,15 @@
-One-shot NAS Algorithms
-=======================
+One-shot NAS
+============
 
-One-shot NAS algorithms leverage weight sharing among models in neural architecture search space to train a supernet, and use this supernet to guide the selection of better models. This type of algorihtms greatly reduces computational resource compared to independently training each model from scratch (which we call "Classic NAS"). NNI has supported many popular One-shot NAS algorithms as following.
+One-shot NAS algorithms leverage weight sharing among models in neural architecture search space to train a supernet, and use this supernet to guide the selection of better models. This type of algorihtms greatly reduces computational resource compared to independently training each model from scratch (which we call "Multi-trial NAS"). NNI has supported many popular One-shot NAS algorithms as following.
 
 
 ..  toctree::
     :maxdepth: 1
 
-    Quick Start <NasGuide>
+    Run One-shot NAS <OneshotTrainer>
     ENAS <ENAS>
     DARTS <DARTS>
-    P-DARTS <PDARTS>
     SPOS <SPOS>
-    CDARTS <CDARTS>
     ProxylessNAS <Proxylessnas>
-    FBNet <FBNet>
-    TextNAS <TextNAS>
-    Cream <Cream>
+    Customize one-shot NAS <WriteOneshot>
```

Comment:
In addition to the out-of-box One-shot algorithms, NNI also provides API for advanced users to define their own one-shot algorithm on NNI. Refer to  **Customize one-shot NAS <WriteOneshot>** for more.